tests/e2e/util.go

```diff
@@ -5791,14 +5794,17 @@ func startCSIPods(ctx context.Context, client clientset.Interface, csiReplicas i
 	ignoreLabels := make(map[string]string)
 	err := updateDeploymentReplicawithWait(client, csiReplicas, vSphereCSIControllerPodNamePrefix,
```

Comment:
Just check if we are reading list_of_pods before performing scaleup/scaledown of CSI controllers pods.